src/app/examples/todos/todos.reducer.spec.ts

```diff
@@ -0,0 +1,23 @@
+import * as fromTodo from './todos.reducer';
+
+describe('TodoReducer', () => {
+  describe('undefined action', () => {
+    it('should return the default state', () => {
+      const { initialState } = fromTodo;
+      const action = {} as any;
+      const state = fromTodo.todosReducer(undefined, action);
+
+      expect(state).toBe(initialState);
+    });
+  });
+
+  describe('ADD action', () => {
+    it('should set loading to true', () => {
```

Comment:
I think this should be something like `Should add a todo`.

Reply:
Ugh, somehow it capitalized should.
I meant to say `should add a todo`.

My bad, sorry....